Sources/NFCPassportReader/PACEHandler.swift

```diff
@@ -61,14 +62,15 @@ public class PACEHandler {
     private var digestAlg : String = ""
     private var keyLength : Int = -1
     
-    public init(cardAccess : CardAccess, tagReader: TagReader) throws {
+    public init(cardAccess : CardAccess, tagReader: TagReader, updateMessageHandler : ((NFCViewDisplayMessage) -> Void?)?) throws {
```

Comment:
I'm curious about the Void-Optional. 😄 Never seen that before. Why do we need it?

Reply:
Good catch, actually we don't need it! 😄 
I think I just copied the closure from here: https://github.com/airsidemobile/NFCPassportReader/blob/4ddd4b2a3e161d48f1c268e91458ecb3a47b31ac/Sources/NFCPassportReader/PassportReader.swift#L69 
Then I changed the return type from String to Void and forgot the delete the Optional. I'll change it 👍